### DISCUSSION TITLE
What's the [F] button in bottom right corner?

### DISCUSSION THREAD

**Discussion by rtabulov** [mpvnet-player/mpv.net#602] [Q&A] [answered]
![image](https://github.com/mpvnet-player/mpv.net/assets/29009741/2ff31bf9-8d88-43e3-98ca-7077a86a4282)
![image](https://github.com/mpvnet-player/mpv.net/assets/29009741/bb7599a7-53bf-4305-b96c-5ee91196fae2)

What does the [ ]  button in bottom right corner do? 
It toggles between [ ] & [F], but what does that mean?

**Comment by stax76** [comment]
I don't know what it means, never saw it before and couldn't find it in the [source code](https://github.com/mpv-player/mpv/blob/master/player/lua/osc.lua).

You can also ask it here:

https://www.reddit.com/r/mpv/

https://github.com/mpv-player/mpv/issues

**Comment by nkh0472** [comment]
LOL that's funny
Is it possible to check what happened in the console when toggle this?

**Comment by Sneakpeakcss** [comment, marked as answer]
That's the very short-lived "[forced-only subtitles](https://www.github.com/mpv-player/mpv/issues/4271)" toggle that was removed few months after it was added [[1]](https://github.com/mpv-player/mpv/commit/945d7c1eda47c97c4bfba884fb21f398a64b2289) -> [[2]](https://github.com/mpv-player/mpv/commit/d0bc90780631f5dceec98da49c30a324cddf64df) -> [[3]](https://github.com/mpv-player/mpv/commit/998c3a163c94748ab413fd01dc77461c7adfc5d7)
